apollo-router/src/plugins/hello.rs

```diff
@@ -1,25 +1,60 @@
 use apollo_router_core::{register_plugin, Plugin};
 use schemars::JsonSchema;
 use serde::Deserialize;
+use std::error::Error;
+use std::fmt;
 use tower::BoxError;
 
-struct Hello {}
+#[derive(Debug)]
+struct HelloError;
 
-#[derive(Default, Deserialize, JsonSchema)]
+impl fmt::Display for HelloError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "HelloError")
+    }
+}
+
+impl Error for HelloError {}
+
+#[derive(Debug)]
+struct Hello {
+    name: String,
+}
+
+impl fmt::Display for Hello {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "Hello")
+    }
+}
```

Comment:
We can delete this part now I guess

Reply:
done: 022587d95a1a1060e126045967cef68efae90c48